.github/workflows/ci.yaml

```diff
@@ -44,4 +44,23 @@ jobs:
       - uses: actions/checkout@v1
       - run: make install
       - run: make zng-output-check
+  test-postgres:
+    runs-on: ubuntu-latest
+    container: golang:1.15
+    services:
+      postgres:
+        image: postgres:11
+        env:
+          POSTGRES_PASSWORD: test
+          POSTGRES_USER: test
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-retries 5
+          --health-timeout 5s
+    steps:
+      - uses: actions/checkout@v1
+      - run: make test-postgres
+        env:
+          PG_TEST: 'postgres://test:test@postgres:5432/test?sslmode=disable'
```

Comment:
Happy to do this on the github side, but my instincts want to keep PG_TEST around: In case I don't want tests interfering with a local postgres instance that I already have running. Django follows a similar pattern.